apps/publikator/components/editor/modules/document/createPasteHtml.js

```diff
@@ -18,13 +18,22 @@ const hasParent = (type, document, key) => {
 const normalise = (html) =>
   html.replace(/<b[^>]*font-weight\s*:\s*normal[^>]*>/g, '')
 
+const PARAGRAPH_TYPES = ['PARAGRAPH', 'INFOP', 'BLOCKQUOTEPARAGRAPH']
+
 export default (centerModule, figureModule) => (event, change, editor) => {
   const transfer = getEventTransfer(event)
-  if (transfer.type !== 'html') return
-
   const cursor = editor.value.selection.anchorKey
   const blockType = editor.value.document.getClosestBlock(cursor).type
 
+  if (transfer.type !== 'html') {
+    if (PARAGRAPH_TYPES.includes(blockType)) {
+      change.insertText(transfer.text.replace(/\n{2,}/g, '\n\n'))
```

Comment:
@annatraussnig re the sentry error: maybe this needs to be `transfer.text?.replace` (also below)?

Reply:
yep i merged a fix for this.